### PR TITLE
Python 3.11 support

### PR DESCRIPTION
**Pull request by corranwebster**
Run pip-based tests on 3.11.  We can't do Wx, PySide2 or PyQt5 easily since there are no wheels.

### CODE REVIEW

**Comment by corranwebster** [comment]
To proceed with this we need either:
- a new celiagg release
- install from celiagg main

File: .github/workflows/bleeding-edge.yml [hunk 1]
```diff
@@ -13,14 +13,13 @@ jobs:
...
-        toolkit: ['null', 'pyside6', 'wx']
-        python-version: ['3.10']
-        exclude:
-          # No Wx wheels available for ubuntu or macos
-          - os: 'ubuntu-latest'
-            toolkit: 'wx'
-          - os: 'macos-latest'
+        toolkit: ['null', 'pyside6']
+        python-version: ['3.11']
+        include:
```
Comment (ankit-gautam23) [inline .github/workflows/bleeding-edge.yml:18]: Why are we including this for this version? Is it needed?

**Review by ankit-gautam23 — CHANGES_REQUESTED** [review]
If it can work with exclude let it be excluded unless there is some points that could be defended.

File: .github/workflows/bleeding-edge.yml (hunk 1 above)
Comment (jwiggins) [reply .github/workflows/bleeding-edge.yml:18]: This ensures that Wx code is still tested by this workflow

Comment (corranwebster) [reply .github/workflows/bleeding-edge.yml:18]: @ankit-gautam23 This is a test suite which runs periodically and alerts Enthought devs if the tests fail due to upstream changes so we can take action.  We want some testing on WxPython, but it isn't easily installable for Python 3.11 yet.

**Comment by corranwebster** [comment]
Tests should pass once https://github.com/celiagg/celiagg/pull/118 is merged

**Review by jwiggins — APPROVED** [review]

**Comment by corranwebster** [comment]
Thanks!

**Pull request merged**